overlord/snapstate/snapstate.go

```diff
@@ -47,9 +47,13 @@ const (
 	// always be enforcing, even if the snap requests otherwise.
 	JailMode
 
-	// if we need flags for just SnapSetup it may be easier
-	// to start a new sequence from the other end with:
-	// 0x40000000 >> iota
+	// flags just for SnapSetup follow
+
+	// HasABA is a feature flag; doInstall sets it, and downstream handlers check for it to handle old SnapSetups
+	HasABA = 0x40000000 >> iota
+
+	// IsRevert flags the SnapSetup as coming from a revert
+	IsRevert
```

Comment:
s/Is// (DevMode above rather than IsDevMode)

Also, can we avoid the surprisingly huge number above? In terms of compatibility, I don't understand why this needs to be large when JailMode above is not and is also new.